03_scaling_out/youtube_face_detection.py

```diff
@@ -119,15 +119,15 @@ def run(url):
     duration = moviepy.editor.VideoFileClip(fn).duration
 
     # Create (start, stop) intervals
-    args = [(fn, offset, offset + 1) for offset in range(int(duration))]
+    intervals = [(fn, offset, offset + 1) for offset in range(int(duration))]
 
-    # Process each range of 1s intervals using a Modal map
-    out_fns = list(detect_faces.starmap(args))
+    print("Processing each range of 1s intervals using a Modal map")
+    out_fns = list(detect_faces.starmap(intervals))
 
-    # Convert to video clips
+    print("Converting detections to video clips")
     out_clips = [moviepy.editor.VideoFileClip(out_fn) for out_fn in out_fns]
 
-    # Concatenate results
+    print("Concatenating results")
```

Comment:
Without this printing, this example can hang with no console output for ~10-15 seconds, which is not so nice.